libs/cart/src/reducers/cart-order-state.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface DaffCartOrderReducerState {
+  order: {id: string | number};
```

Comment:
seems weird to make this an object, but not make it generic. I think we should go with one or the other. Probably make it generic to future proof it.

Reply:
Making it generic would introduce a dependency onto checkout. Maybe thats a reason to move the place order driver into checkout instead.

Reply:
changed